mxcubeweb/routes/login.py

```diff
@@ -66,8 +55,7 @@ def signout():
         Signout from MXCuBE Web and reset the session
         """
         app.usermanager.signout()
-
-        return redirect("/login", code=302)
+        return make_response(jsonify(""), 200)
```

Comment:
Maybe add a meaningful message instead of empty string:

```suggestion
        return make_response(jsonify("Successful sign out"), 200)
```

Reply:
OK, after response from @axelboc I understand we could leave it as empty for now, until we get to the point of being able to return the actual login info instead.